tsl/test/expected/dist_hypertable-12.out

```diff
@@ -535,22 +535,19 @@ SELECT * FROM disttable;
          ->  Custom Scan (DataNodeScan) on public.disttable disttable_1
                Output: disttable_1."time", disttable_1.device, disttable_1.temp
                Data node: db_dist_hypertable_1
-               Fetcher Type: Row by row
```

Comment:
Why did you remove fetcher type from explain output?

Reply:
Now the fetcher type is ultimately determined at the execution stage, so we can show it only in EXPLAIN ANALYZE. I'm not sure, maybe we should do this at planning stage? It is relatively heavy and involves looking up the send/receive functions for all participating columns, so I kept it at execution stage for now.